src/ontology/mondo-edit.obo

```diff
@@ -201149,17 +201149,14 @@ synonym: "congenital hypothalamic hamartoma syndrome" EXACT [OMIM:241800]
 synonym: "hamartoma of hypothalamus" EXACT [NCIT:C4385]
 synonym: "hamartoma of the hypothalamus" RELATED [GARD:0002934]
 synonym: "hypothalamic hamartoma" EXACT [NCIT:C4385]
-synonym: "hypothalamic hamartomas" EXACT [OMIM:241800]
-synonym: "Pallister-Hall-like syndrome" EXACT [OMIM:241800, OMIM:genemap2]
+synonym: "hypothalamic hamartomas" EXACT [OMIM:146510]
 xref: GARD:2934 {source="MONDO:GARD"}
 xref: ICD9:759.6 {source="MONDO:relatedTo", source="MONDO:i2s"}
-xref: MEDGEN:137970 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
```

Comment:
Yes, I removed this in [this commit](https://github.com/monarch-initiative/mondo/pull/7879/commits/f4314a69fbb40e40341ea8ca5961c63c04e62ba7). The [definition](https://www.ncbi.nlm.nih.gov/medgen/?term=137970) mentions Pallister-Hall-like syndrome (PHLS). It is difficult to separate out `Pallister-Hall-like syndrome` from `Pallister-Hall syndrome`. However, Pallister-Hall syndrome is caused by a mutation in [GL13 based on OMIM](https://omim.org/entry/146510?search=146510&highlight=146510) so I tried to adjust the properties here for Pallister-Hall syndrome.

Reply:
Following our conversation, I'll add back these properties (synonyms and xrefs) since it was decided this term is supposed to represent PALLISTER-HALL-LIKE SYNDROME and it's the axiom `has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4319 {source="MONDO:mim2gene_medgen"} ! GLI3` that is incorrect and should be `SMO`.